diff --git a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheet/BottomSheet.tsx b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheet/BottomSheet.tsx
index b166440..5ceb490 100644
--- a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheet/BottomSheet.tsx
+++ b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheet/BottomSheet.tsx
@@ -1561,7 +1561,6 @@ const BottomSheetComponent = forwardRef<BottomSheet, BottomSheetProps>(
       _providedIndex,
       animatedCurrentIndex,
       isAnimatedOnMount,
-      handleSnapToIndex,
     ]);
     //#endregion
 